ko/lessons/specifics/ecto.md

```diff
@@ -17,12 +26,10 @@ defp deps do
 end
 ```
 
-이제 Ecto와 어댑터를 application의 목록에 추가할 수 있습니다.
+아래 명령어를 통해 의존성 있는 라이브러리를 가져옵니다:
```

Comment:
We don't usually use colons in native Korean writings, how about changing to periods?